docs/pages/database-access/guides/aws-cross-account.mdx

```diff
@@ -0,0 +1,196 @@
+# in us-west-1 within AWS account `222222222222` by assuming an external AWS
+# IAM role "example-role".
+db_service:
+  enabled: "yes"
+  databases:
+  - name: "rds-postgres"
+    protocol: "postgres"
+    uri: "rds-postgres.abcdef012345.us-west-1.rds.amazonaws.com:5432"
+    aws:
+        assume_role_arn: "arn:aws:iam::222222222222:role/example-role"
+        external_id: "example-external-id"
+```
+
+Restart the Teleport Database Service for the configuration file changes to take
+effect.
+
+</TabItem>
+<TabItem label="Dynamic Config">
+Create a dynamic database resource to dynamically register an AWS database
+in an external account and proxy connections to it.
+
+```yaml
+# This example configuration will dynamically register an RDS PostgreSQL instance
+# in us-west-1 within AWS account `222222222222`.
+# Teleport Database Service agents that match its labels with resource selectors
+# will proxy the database by assuming the configured external AWS IAM role.
+kind: db
+version: v3
+metadata:
+  name: "rds-postgres"
+  description: "Example dynamic database resource"
+  labels:
+    env: "dev"
+spec:
+  protocol: "postgres"
+  uri: "rds-postgres.abcdef012345.us-west-1.rds.amazonaws.com:5432"
+  aws:
+    # Note that account_id must match the AWS account ID in `assume_role_arn`.
+    # Dynamic database resources do not derive `account_id` from
+    # `assume_role_arn` automatically (unlike static configuration).
+    account_id: "222222222222"
+    assume_role_arn: "arn:aws:iam::222222222222:role/example-role"
+    external_id: "example-external-id"
+```
+
+Save the configuration to a file like `database.yaml` and create it with `tctl`:
+```code
+$ tctl create database.yaml
+```
+For more information about database registration using dynamic database
+resources, see: [Dynamic Registration](dynamic-registration.mdx).
+
+</TabItem>
+</Tabs>
+
+(!docs/pages/includes/aws-credentials.mdx service="the Database Service"!)
+
+## AWS IAM permissions policies
+
+AWS IAM policies must be configured for both the Teleport Database Service's
+AWS IAM identity and the external AWS IAM role:
+- The Teleport Database Service's AWS IAM identity must have permission to
+  assume the external role.
+- The external AWS IAM role's trust policy must trust the Teleport Database
+  Service's AWS IAM identity.
+
+<Details title="Why are both required?">
+  Unlike assuming a role within the same AWS account, when an AWS IAM role is
+  in a different AWS account than the IAM identity that attempts to assume it,
+  the role's trust policy alone is not sufficient to allow assuming the role.
+  
+  For more details, see:
+  [AWS cross-account policy evaluation](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_evaluation-logic-cross-account.html#policy-eval-cross-account).
+</Details>
+
+(!docs/pages/includes/database-access/aws-bootstrap.mdx attachToRole="teleport-db-service" attachToUser="teleport-db-service"!)
```

Comment:
The text of this partial ("Teleport can bootstrap...") makes it sound like bootstrapping IAM permissions using `teleport db configure bootstrap` is optional.

I think this section should be more firm in the expectations we have of IAM policies. Are we saying that users _must_ attach the policies resulting from `db configure bootstrap` to the Database Service's IAM identity?

I also think it would be good to include an example of the IAM policies we require so it's clearer to users what we expect.

Reply:
I added an example permission policy specific to granting `AssumeRole` access, which is the focus of this guide: d818b6114c3eff4df981b7eabed2732f0121453e

The `bootstrap` command can be used to print or automatically add permissions for all the different databases in a config file, including adding the `sts:AssumeRole` permission where needed. Setting up permissions using the automatic bootstrap is optional, but configuring permissions is of course not optional - we need those permissions for various things that the db service will do.

I do state above "The Teleport Database Service's AWS IAM identity must have permission to assume the external role.". Let me know if we should refine this further